CHANGELOG

```diff
@@ -4,6 +4,7 @@
   - Revision history: Improve version history for content with carriage return
   - Tylium: Show liquid content in editor preview
   - Web-server: Replace unicorn with puma in production
+  - Activity: Filter activities by user, type and date
```

Comment:
```suggestion
  - Activity: Filter activities by user, type, and date
```

Reply:
Should we add a comma when we have the and right after it?

Reply:
I generally add it in a list context. Without the comma, this can be interpreted as "you can filter by user and/or type + date", suggesting that type and date are always done together. With the comma, it avoids ambiguity by expressing that filtering can be done by any combination of the 3 options. This is just a #suggestion and my personal style of writing. I don't feel strongly about it but Grammarly was correcting it so I added it as a comment.

Reply:
I see how it could create confusion. Changed it!